CONTRIBUTING.md

```diff
@@ -1,8 +1,45 @@
-# Guide to contribute to the Stream Chat Java SDK
-## Requirements
+# :recycle: Contributing <!-- omit in toc -->
+
+## Requirements <!-- omit in toc -->
 To contribute to this project, you need to have [Lombok](https://projectlombok.org/) installed in your IDE.
 You also need to use javac >= 11 to compile (ensure that your JAVA_HOME points to such a location), even if the produced library is compatible with Java 8 (for some reasons, jdk 8 compiler fails on some Lombok annotations).
 
+- [Setup the project](#setup-the-project)
+- [Architecture explanation](#architecture-explanation)
+  - [Model classes](#model-classes)
+  - [Request classes](#request-classes)
+  - [RequestData and RequestObject classes](#requestdata-and-requestobject-classes)
+  - [Response classes](#response-classes)
+  - [Service interfaces](#service-interfaces)
+  - [The StreamServiceGenerator class](#the-streamservicegenerator-class)
+  - [The StreamResponse interface and related classes](#the-streamresponse-interface-and-related-classes)
+  - [The StreamRequest and StreamServiceHandler classes](#the-streamrequest-and-streamservicehandler-classes)
+  - [Test classes](#test-classes)
+- [Code rules](#code-rules)
+- [How to](#how-to)
+  - [Enable logging](#enable-logging)
+  - [Add a new endpoint with request body schema or json query parameter](#add-a-new-endpoint-with-request-body-schema-or-json-query-parameter)
+    - [Create the Model and Service](#create-the-model-and-service)
+    - [Create the RequestObject classes](#create-the-requestobject-classes)
+    - [Create the xxxResponse class](#create-the-xxxresponse-class)
+    - [Create the xxxRequestData class](#create-the-xxxrequestdata-class)
+    - [Create the xxxRequest](#create-the-xxxrequest)
+    - [Add the endpoint path variables to the xxxRequest class](#add-the-endpoint-path-variables-to-the-xxxrequest-class)
+    - [Add the request creation method](#add-the-request-creation-method)
+    - [Add the service method](#add-the-service-method)
+    - [Make the xxxRequest extend StreamRequest<xxxResponse>](#make-the-xxxrequest-extend-streamrequestxxxresponse)
+    - [Update README.md (features and examples)](#update-readmemd-features-and-examples)
+  - [Add a new endpoint without request body schema or json query parameter](#add-a-new-endpoint-without-request-body-schema-or-json-query-parameter)
+    - [Create the Model and Service](#create-the-model-and-service-1)
+    - [Create the xxxResponse class](#create-the-xxxresponse-class-1)
+    - [Create the xxxRequest](#create-the-xxxrequest-1)
+    - [Add the request creation method](#add-the-request-creation-method-1)
+    - [Add the service method](#add-the-service-method-1)
+    - [Make the xxxRequest extend StreamRequest<xxxResponse>](#make-the-xxxrequest-extend-streamrequestxxxresponse-1)
+    - [Update DOCS.md (features and examples)](#update-docsmd-features-and-examples)
+  - [Add new fields to requests or responses](#add-new-fields-to-requests-or-responses)
```

Comment:
I'm going to be honest. I understand someone spent a lot of time writing these, but I don't think it's needed. It's super long, no one will read it, and you'll understand the code if you take a look at it yourself. Would you guys agree removing it?

Reply:
agreed